lib/Sema/CodeSynthesis.cpp

```diff
@@ -1304,15 +1319,15 @@ HasMemberwiseInitRequest::evaluate(Evaluator &evaluator,
   llvm::SmallPtrSet<VarDecl *, 4> initializedProperties;
   llvm::SmallVector<std::pair<VarDecl *, Identifier>> invalidOrderings;
 
-  for (auto *member : decl->getMembers()) {
```

Comment:
Oh, I think this method should also use `decl->getImplementationContext()->getAllMembers()` here.